src/hw_capemgr.js

```diff
@@ -20,9 +21,9 @@ exports.readPWMFreqAndValue = function (pin, pwm) {
         mode.value = duty / period;
     } catch (ex) {}
     return (mode);
-};
+},
```

Comment:
Gah. I guess it is OK, just seems unnecessary to me to put function definitions into an object definition, rather than defining things a function at a time. Feels a bit squirrelly to me to end function definitions with a comma.

Reply:
Do i need to make the module.exports to be separate from the function definitions ?